### ISSUE TITLE
About the DeepRT+ calibration

### ISSUE THREAD

**Issue by Chaohk** [horsepurve/DeepRT#13]
Hi, could you please tell me that how the DeepRT+ do the calibration using a certain ratio of the test-group peptides after pretraining with the big data?

Thanks.

**Comment by horsepurve** [comment]
Hi! For the calibration from a pretrained model (run1) to another LC run (run2), please:

1. Select a certain ratio of peptides from run2 as the calibration peptides (better with high identification confidence, and span across the whole LC run). Save as a txt file and assign its file path to "train_path" in "config.py".
2. Set "pretrain_path" to the file path of your pretrained model, set "dict_path" and "max_length" to be the same with those of the pretrained model (see [tutorial](https://github.com/horsepurve/DeepRTplus#4-transfer-learning-using-our-pre-trained-models) for details).
3. Run by "python capsule_network_emb.py" and see if the prediction is improved and better than not using a model pretrained from bigger data.

**Comment by Chaohk** [comment]
Thank you for telling me the usage of the calibration function. Could I know what algorithm and methods are using in the RT calibration. Like piecewise linear regression?

**Comment by horsepurve** [comment]
The method is **transfer learning**. The model is first **pre-trained** on data1 from LC run1, then **fine-tuned** on data2 from LC run2.

The only difference between using/not using this pretraining & fine-tuning scheme is that: without it, we train a deep neural network (DNN) from scratch whose weights are randomly initialized; whereas pretraining provides a well-educated guess for the weight of DNN so that DNN can converge to a better minimum faster by stochastic gradient descent.

This pretraining & fine-tuning routine is extremely widely adopted in Computer Vision, Natural Language Processing, etc. In LCMS, our work is perhaps the first that uses this to integrate/calibrate multiple runs across different LCs/experiments/labs.

**Comment by Chaohk** [comment]
Fully understand. Thanks a lot!
